pages/blog/integrating-mailchimp-with-your-nodejs-app.mdx

```diff
@@ -0,0 +1,350 @@
+---
+title: "Integrating MailChimp with your Node.js App"
+date: "2021-06-01"
+author: "Yashovardhan Agrawal"
+pullQuote: "A step by step guide of integrating your MailChimp Audiences with your Node.js app"
+image: "integrating-mailchimp-with-your-nodejs-app/mc4.png"
+tags: [connections, engineering]
+---
+
+# Integrating MailChimp with your Node.js App
```

Comment:
This is unnecessary, and we'll get dinged in SEO points for having it, as the title is already rendered as an `<h1>` at the top of the page. [See preview here](https://www-grouparoo-git-fork-yashovardhan-mailchimp-nodejs-gr-514bff.vercel.app/blog/integrating-mailchimp-with-your-nodejs-app).

Reply:
thanks for pointing it out, I've removed the heading.